frontend-react/src/pages/how-it-works/SystemsAndSettings.tsx

```diff
@@ -50,7 +50,7 @@ export const SystemsAndSettings = () => {
                 <p>ReportStream can send a variety of data types to a public health department test environment to make sure everything is working correctly before launch.</p>
 
                 <ul>
-                    <li><span className="text-bold">Synthetic data (standard):</span> ReportStream always sends synthetic data in our <a href="https://github.com/CDCgov/prime-data-hub/blob/production/prime-router/docs/schema_documentation/primedatainput-pdi-covid-19.md&sa=D&source=editors&ust=1619546431803000&usg=AOvVaw1E09LDMJ0kGE2qa_OPZojW" className="usa-link">standard data schema</a> for your review.</li>
+                    <li><span className="text-bold">Synthetic data (standard):</span> ReportStream always sends synthetic data in our <a href="https://github.com/CDCgov/prime-reportstream/blob/production/prime-router/docs/schema_documentation/primedatainput-pdi-covid-19.md" target="_blank" className="usa-link">standard data schema</a> for your review.</li>
```

Comment:
links with `target="_blank"` should always include a `rel="noreferrer"` to prevent "[Reverse Tabnapping](https://owasp.org/www-community/attacks/Reverse_Tabnabbing)"

Reply:
Great catch, @TomNUSDS! I've pushed that change to this branch as of this morning. :)